React/Views/RCTRefreshControl.m

```diff
@@ -31,6 +32,18 @@ - (instancetype)init
 - (void)layoutSubviews
 {
   [super layoutSubviews];
+  
+  // UIRefreshControl is managed so avoid changing it directly
+  // This method may break in future versions of iOS (subview structure may change)
+  // And self.transform won't shift title text
+  if(_isInitialRender) {
```

Comment:
Also, cc @brentvatne since it sounded like you were interested in this PR. Can you take a look?